src/lasp_dependence_dag.erl

```diff
@@ -205,16 +211,31 @@ handle_call({will_form_cycle, From, To}, _From, #state{dag=Dag}=State) ->
 %%
 %%      We monitor all edge Pids to know when they die or get restarted.
 %%
-handle_call({add_edges, Src, Dst, Pid}, _From, #state{dag=Dag, process_map=Pm}=State) ->
-    %% @todo Add metadata and duplicate checking.
-    Status = [digraph:add_edge(Dag, V, Dst, Pid) || V <- Src],
+handle_call({add_edges, Src, Dst, Pid, [ReadFuns, TransFun, {Dst, WriteFun}]},
+            _From, #state{dag=Dag, process_map=Pm}=State) ->
+
+    %% @todo Add duplicate checking.
```

Comment:
Can you give me an example where a duplicate would occur?  Is it just if someone literally runs the same command multiple times?